egs/wsj/s5/steps/cleanup/segment_long_utterances.sh

```diff
@@ -240,15 +281,14 @@ if [ $stage -le 7 ]; then
       --input-idf-stats=$dir/docs/idf_stats.txt \
       $sdir/docs.JOB.txt $sdir/src_tf_idf.JOB.txt 
 
+  sdir=$dir/docs/split$nj
   # Make $sdir an absolute pathname.
   sdir=`perl -e '($dir,$pwd)= @ARGV; if($dir!~m:^/:) { $dir = "$pwd/$dir"; } print $dir; ' $sdir ${PWD}`
 
   for n in `seq $nj`; do
     awk -v f="$sdir/src_tf_idf.$n.txt" '{print $1" "f}' \
       $sdir/text2doc.$n
-  done > $dir/docs/source2tf_idf.scp
-
-
+  done | perl -ane 'BEGIN{ %tfidfs = (); } { if (!defined $tfidfs{$F[0]}) { $tfidfs{$F[0]} = $F[1]; } } END{while(my ($k, $v) = each %tfidfs) { print "$k $v\n"; }}' > $dir/docs/source2tf_idf.scp
```

Comment:
break up this line-- doesn't have to be 80 chars but don't make it over 100 or so.